### ISSUE TITLE
Possibility of nut retrieving current dependencies?

### ISSUE THREAD

**Issue by ttacon** [jingweno/nut#1]
Would it be possible to add a subcommand that locks down the current versions of dependencies instead of having to enter them manually in the toml file (sorry if this exists and I missed it)?

Meaning, you could pass the output of `go list -f "{{.Imports}}" ./...` or something similar to something along the lines of `git rev-parse --verify --short HEAD` and these would be automatically inserted into the toml file?

Just a thought for making getting started with nut on an existing project easier! Cool project by the way!

**Comment by owenthereal** [comment]
@ttacon Yup, that's a good idea. I also plan to provide a migration path from tools like `godep` etc.

**Comment by ttacon** [comment]
Sweet! If I get any time this week I'll try to help out, this is a tool I'd like to use but I don't want to have to manually fill out the toml file (I'm lazy) for all my projects.
